app/app/src/main/java/com/example/myapplication/MainActivity.java

```diff
@@ -149,32 +163,34 @@ public void onClick(View view) {
         });
 
         // Click listeners for changing activity content
-        joystickContentBtn.setOnClickListener(new View.OnClickListener() {
-            @Override
-            public void onClick(View v) {
-                if (headsetIsConnected && carIsConnected) {
-                    joystickContent.setVisibility(View.VISIBLE);
-                    eegContent.setVisibility(View.GONE);
-
-                    joystickContentBtn.setVisibility(View.GONE);
-                    eegContentBtn.setVisibility(View.VISIBLE);
-
-                    if (eegActive) {
-                        eegActive = false;
-                        stopEeg();
-                        stopGyro();
-                    }
-                }
-                else {
-                    // do nothing
-                }
-            }
-        });
 
-        eegContentBtn.setOnClickListener(new View.OnClickListener() {
+      joystickContentBtn.setOnClickListener(new View.OnClickListener() {
+          @Override
+          public void onClick(View v) {
+           //   if (headsetIsConnected && carIsConnected) {
+//                    //joystickContent.setVisibility(View.VISIBLE);
+//                    eegContent.setVisibility(View.GONE);
+//
+//                    joystickContentBtn.setVisibility(View.GONE);
```

Comment:
This line changes the appearance of the button in the UI that lets end-users know that they have switched the content of the app UI. It needs to be included and cannot be commented out, in order for the header to work. Is it represented somewhere else instead?

Reply:
There is a problem. I cannot switch to the second activity if I remain those. Will try to remove second activity to main activity to see if it works

Reply:
That is because it was set-up to only dispaly and hide content in a single activity.